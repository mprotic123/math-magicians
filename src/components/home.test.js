import React from 'react';
import renderer from 'react-test-renderer';
import Home from './home.js';

it('renders home', () => {
  const home = renderer.create(
    <Home />,
  ).toJSON();
  expect(home).toMatchSnapshot();
});