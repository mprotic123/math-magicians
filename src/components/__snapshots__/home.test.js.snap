// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders home 1`] = `
<div
  className="home"
>
  <h1>
    Welcome to this webApp!
  </h1>
  <p>
    Lorem ipsum dolor sit amet, consectetur adipisicing elit. Odio eum labore sequi. Qui dolorum enim deserunt minima ratione, consequuntur quas esse culpa aliquam rem, accusantium consectetur labore odio repellat consequatur?
  </p>
  <p>
    Lorem ipsum dolor sit amet, consectetur adipisicing elit. Odio eum labore sequi. Qui dolorum enim deserunt minima ratione, consequuntur quas esse culpa aliquam rem, accusantium consectetur labore odio repellat consequatur?
  </p>
</div>
`;
